### ISSUE TITLE
Feature request: add RegExps support for client domain

### ISSUE THREAD

**Issue by quytx** [MatthewLarner/cross-domain-storage#29]
Hi Matthew, I'm wondering if we could add support for regular expression when defining client domain? 
Thanks!

**Comment by MatthewLarner** [comment]
Yep, great idea. I'll try and put this together in the next couple of days.
Cheers
